Documentation/metering-architecture.md

```diff
@@ -118,9 +118,9 @@ When a `Report` is created, and it sees the creation event, it does the followin
 - Execute the query using Presto.
 - Update the Report status that everything succeeded.
 
-[presto-overview]: https://prestodb.io/docs/current/overview/use-cases.html
+[presto-overview]: https://prestosql.io/docs/current/overview/use-cases.html
```

Comment:
@martint, curious where you get this link from?  The current link seems to be what is used by the official github page. `http://prestodb.github.io/`

Reply:
@pruan-rht you're looking at the old github. Presto has moved to it's own foundation and Martin is a founder of Presto whom made the transition happen.

Reply:
A simple mouse-over would've shown `Creator of Presto` on his name...sorry about that @martint